app/utilities/trackEvent.ts

```diff
@@ -0,0 +1,16 @@
+type GTagEvent = {
+  action: string;
+  category?: string;
+  label?: string;
+  value?: number;
+};
+
+const trackEvent = ({ action, category, label, value }: GTagEvent): void => {
+  window.gtag("event", action, {
```

Comment:
_Probably_ want to do a `window` undefined check here just in case ;)

Reply:
Whoopsie sorry I just merged it last night. Want to create a new PR and we can chuck it in before the button goes up later today?

Reply:
That's all good! It's probably being extra defensive 💃 but yeh will do 1 sec